### DISCUSSION TITLE
Search by text

### DISCUSSION THREAD

**Discussion by julianalsemmani** [requarks/wiki#5438] [Help / Questions] [answered]
Is there a possibility to search by the text in the wiki-article?

Since, now It's only possible to search by title and keywords as I see it.

Is this a possible functionality to add?

**Comment by julianalsemmani** [comment, marked as answer]
Okey, so by looking a bit around, I found out that the "Short Description" of the article is searchable.

**Reply by vuducmanh11** [reply]
You can enable the other default searchengine in admin page to full-text search.
https://docs.requarks.io/search/postgres

**Reply by julianalsemmani** [reply]
Thank you! Didn't quite know what the "Search Engine" in the admin page was for.
